Examples/object_detection/ObjectDetection/ModelDataHandler/ModelDataHandler.swift

```diff
@@ -14,6 +14,7 @@
 
 import CoreImage
 import TensorFlowLite
```

Comment:
Do example still need this? If not, remove this help to understand this example.

Reply:
@tucan9389 
Your review is really helpful. Thank you. I'll remove it.